File: web/app/model/code/Team.js
Ext.define('iFlat.model.code.Team', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'id', mapping: 'id', type: 'string'},
        {name: 'team.deptCode', mapping: 'deptCode', type: 'string'},
        {name: 'team.deptName', mapping: 'deptName', type: 'string'},
        {name: 'team.teamName', mapping: 'teamName', type: 'string'},
        {name: 'team.teamFullName', mapping: 'teamFullName', type: 'string'},
        {name: 'team.type', mapping: 'type', type: 'string'},
    ]
});

File: src/main/java/com/iflat/code/bean/Team.java
package com.iflat.code.bean;

/**
 * Created by tyriv on 2016/1/18.
 */
public class Team {

    private String deptCode;
    private String deptName;
    private String teamName;
    private String type;
    private String teamFullName;

    public String getTeamFullName() {
        return teamFullName;
    }

    public void setTeamFullName(String teamFullName) {
        this.teamFullName = teamFullName;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getDeptCode() {
        return deptCode;
    }

    public void setDeptCode(String deptCode) {
        this.deptCode = deptCode;
    }

    public String getDeptName() {
        return deptName;
    }

    public void setDeptName(String deptName) {
        this.deptName = deptName;
    }

    public String getTeamName() {
        return teamName;
    }

    public void setTeamName(String teamName) {
        this.teamName = teamName;
    }
}
